<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>790</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame_how_it_works">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="horizontalSpacing">
       <number>5</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_info">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>15</height>
         </size>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;b&gt;Jak to dzia≈Ça?&lt;/b&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_info_icon">
        <property name="maximumSize">
         <size>
          <width>15</width>
          <height>15</height>
         </size>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>i</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout_5">
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Warstwa z projektu</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer_8">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Kolumna TERYT</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_info_column">
          <property name="maximumSize">
           <size>
            <width>15</width>
            <height>15</height>
           </size>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>i</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Nazwa warstwy
wynikowej</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="text_edit_layer_name">
        <property name="placeholderText">
         <string>nowa_warstwa</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QgsMapLayerComboBox" name="layer_select"/>
      </item>
      <item row="3" column="1">
       <widget class="QgsFieldComboBox" name="combobox_teryt_column"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="1" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>15</number>
        </property>
        <item>
         <widget class="QLabel" name="label_status">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_found_count">
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="indent">
           <number>6</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_not_found_count">
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="indent">
           <number>6</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QFormLayout" name="formLayout_6">
          <item row="0" column="0">
           <widget class="QPushButton" name="button_start">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_info_start">
            <property name="maximumSize">
             <size>
              <width>15</width>
              <height>15</height>
             </size>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="text">
             <string>i</string>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="button_cancel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Anuluj</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progress_bar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>0</height>
      </size>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="table_errors"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_save_not_found">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Zapisz nieznalezione</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFieldComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsfieldcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
